index/upsidedown/upsidedown.go

```diff
@@ -135,22 +135,22 @@ func (udc *UpsideDownCouch) loadSchema(kvreader store.KVReader) (err error) {
 }
 
```

Comment:
This file have unrelated changes coming from go fmt, 
we have another PR raised for "go fmt" changes (https://github.com/blevesearch/bleve/pull/1874/) based off of master.
We can rebase this PR on top of that.

Reply:
This is just a go fmt and given that it'll be backported as is - let's leave this as is for now and rebase #1874 instead.